### ISSUE TITLE
Fable.Remoting.Json no longer deserializes Set

### ISSUE THREAD

**Issue by Shmew** [Zaid-Ajaj/Fable.Remoting#173]
After updating to 2.9.0 a DU that looks like this:

```fsharp
type SomeOtherDU =
    | SomeOtherCase
    
type MyDU =
    | SomeCase
    | CustomCase of Set<SomeOtherDU>
```

Gives an error of:
```bash
Cannot convert "SomeOtherCase" to ["Set",null]
```

**Comment by Zaid-Ajaj** [comment]
Do you get this error from the server side or from the client side of things? 

> It looks like a SimpleJson error on the front-end side of things

I will check it out

**Comment by Shmew** [comment]
It's on the front-end, so some sort of mis-match between Fable.SimpleJson and Fable.Remoting.Json

**Comment by Zaid-Ajaj** [comment]
Fixed in SimpleJson v3.10.1 and updated v6.6.1 subsequently to include the SimpleJson fix
